src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,47 @@
+package tests;
+import com.codeborne.selenide.Configuration;
+import org.testng.annotations.*;
+import util.TestListener;
+import helpers.ConfigurationManager;
+import org.openqa.selenium.WebDriver;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+@Listeners({TestListener.class})
+public abstract class BaseTest {
+    protected final int WAIT_TIMEOUT_SECONDS =  Integer.parseInt(ConfigurationManager.getProperty("webdriver.timeout.seconds"));
+
+    static {
+        File allureResults = new File("target/allure-results");
+        if(allureResults.isDirectory()) {
+            for (File item: Objects.requireNonNull(allureResults.listFiles())) {
+                item.delete();
+            }
+        }
+    }
+    @BeforeMethod
+    public void setUp()  {
+        Configuration.browser = "chrome";
+        Configuration.browserSize = "1920x1080";
```

Comment:
please remove hardcoded values and use env-dependent properties. For driver settings you can specify -Dselenide.browser=chrome